docs/changelog-fragments.d/173.bugfix.md

```diff
@@ -0,0 +1,3 @@
+Disable ANSI escapes when running ansible and ansible-lint to prevent failure
```

Comment:
Would this be more precise? It should be in the past tense or otherwise, suggest that it's changed since the previous release.
```suggestion
Turned off escaping ANSI codes when running ansible and ansible-lint to prevent failure
```